whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/client/NrtmRestClient.java

```diff
@@ -89,9 +89,10 @@ public String getNotificationFileSignature(final String source){
     }
 
     @Nullable
-    public byte[] getSnapshotFile(final String url){
+    public byte[] getSnapshotFile(final String url, final String source){
         try {
```

Comment:
Instead of using baseUrl can you use it from UNF url and add relative url path? This will laos make sure that we are forming the correct path and this will break in case url is not properly formed.

Reply:
in the UNF we only have the snap/delta relative path, we don't have the UNF url anywhere